.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: ''
+assignees: ''
+
+---
+
+- [] I have checked that this bug exists on the latest version of AutoGluon
```

Comment:
"the latest stable release version"

also add a new checkbox:

`- [] and/or I have checked that this bug exists on the latest mainline of AutoGluon via source installation`